coscale-dev-env/deploy-agent.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I added a reference to this script in the README, it is not used in the deploy but can be used afterwards to install a CoScale agent on all VMs in a resource group.